src/tablecloth/time/api/rolling_window.clj

```diff
@@ -86,9 +86,17 @@
 ;; support alternate approaches to build grouped datasets for rolling-window
 
 
+;; (defn rolling-window
+;;   "rolling-window dataset"
+;;   [ds column-name window-size]
+;;   (tablecloth/group-by ds (rw-map ds column-name window-size)))
+
+
 (defn rolling-window
-  "rolling-window dataset"
-  [ds column-name len]
-  (tablecloth/group-by ds (rw-map ds column-name len)))
+  "rolling-windows for a dataset"
```

Comment:
Since this is a public api fn, consider adding a bit more of a doc string here? Something like:
```suggestion
  "Groups data into windows of `window-size`. Returns a
  grouped dataset that can be used with `tablecloth.api.aggregate`,
  for example."
```
Or something like that. Not sure how to put it...